src/modules/super-resolution/includes/Attachment_Field.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Wordlift\Modules\Super_Resolution;
+
+class Attachment_Field {
+
+	public function register_hooks() {
+		add_filter(
+			'attachment_fields_to_edit',
+			array( $this, 'attachment_fields_to_edit' ),
+			10,
+			2
+		);
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+
+	}
+
+	/**
+	 * @param $form_fields
+	 * @param \WP_Post    $post The WP_Post attachment object.
+	 *
+	 * @return mixed
+	 */
+	public function attachment_fields_to_edit( $form_fields, $post ) {
+		// Add your custom HTML code here
+		$form_fields['wl_super_resolution'] = array(
+			'label'  => __( 'WordLift Image Upscale', 'wordlift' ),
+			'input'  => 'custom',
+			'html'   => '',
+			'custom' => $this->get_html( $post->ID ),
+		);
+
+		return $form_fields;
+	}
+
+	public function admin_enqueue_scripts() {
+		/**
+		 * I only need these files when the media library is rendered.
+		 */
+		if ( ! did_action( 'wp_enqueue_media' ) ) {
+			return;
+		}
+
+		wp_enqueue_style( 'wl-super-resolution', WL_DIR_URL . 'modules/super-resolution/css/super-resolution.css', array( 'thickbox' ), WORDLIFT_VERSION );
+		wp_enqueue_script( 'wl-angular-app' );
+
+	}
+
+	private function get_html( $attachment_id ) {
+		$base_url                      = WL_ANGULAR_APP_URL . "#(dialog:dialogs/super-resolution/$attachment_id/upscale)";
```

Comment:
@naveen17797 I think the dashboard layout should be the default one because dialogs can also open on it.